frontend/web/package.json

```diff
@@ -13,6 +13,7 @@
     "test": "jest",
     "lint": "eslint --ext .js,.ts,.tsx ./",
     "lint-fix": "eslint --fix --ext .js,.ts,.tsx ./",
+    "style-fix": "prettier --write .",
```

Comment:
Suggestion: Can we name this `fmt`? That way all formatting in this project use the same command (like `go fmt`.